pnpm-lock.yaml

```diff
@@ -648,7 +648,7 @@ packages:
       rollup: 2.41.1
     dev: false
 
-  /@sveltejs/vite-plugin-svelte/1.0.0-next.7_301da3dca3251a36e99a1622a4d7ca38:
+  /@sveltejs/vite-plugin-svelte/1.0.0-next.7_6c43afb327f3ef976774a1a1e8559c8a:
```

Comment:
no clue what that is about, i'm using pnpm 6.2.1, but that hash should not have changed? wonder if its a local cache thing.